.gitignore

```diff
@@ -21,6 +21,7 @@ artifacts
 /certs
 # make stress, acceptance produce stress.test, acceptance.test
 *.test*
+pkg/cmd/roachtest/_runner-logs
```

Comment:
How did you get them there? :)

Reply:
Hmm, they just pop up whenever I run a roachtest unit test that calls create. It's been happening for as long as I can remember but perhaps it's just me? I can remove it if it's just me.